contrib/kube-prometheus/assets/prometheus/rules/etcd3.rules

```diff
@@ -0,0 +1,177 @@
+# general cluster availability
```

Comment:
Are those the ones the etcd team has defined for k8s etcd or the general etcd3 ones?